elastic/integrations
# IIS beta to ga

## What does this PR do?
Moving the IIS integration from beta to GA. 



## Checklist




## How to test this PR locally
+ Install IIS server in the windows platform
+ Add IIS agent integration
+ Check whether the collect logs and metrics option is enabled
+ Verify the data is ingested into Elasticsearch
+ Check dashboards loads with the data and generates metrics

Since IIS is windows platform specific integration. Manual testing is performed and system testing is not added.



## Related issues

- Closes #3523
